include/Reader/readerir.h

```diff
@@ -1317,6 +1325,7 @@ class GenIR : public ReaderBase {
   uint32_t TargetPointerSizeInBits;
   const uint32_t UnmanagedAddressSpace = 0;
   const uint32_t ManagedAddressSpace = 1;
+  static uint32_t TailCallCount;
 };
```

Comment:
I can't understand how JIT64 preserved the thread safety.  They do have the code #ifdefd out for Release but that seems to be all I can see.